### PR TITLE
Remove `run_command` utility function

### PR DESCRIPTION
**Pull request by portante**

### CODE REVIEW

**Review by webbnh — APPROVED** [review]
It's truly a thing of beauty.

**Review by dbutenhof — DISMISSED** [review]
Yeah, that works.

#2741 is going to be a moment ... the merge seems to have broken our favorite test-61 and I need to figure out why... which of course is complicated by the fact that I moved all the code and had to do the rebase cleanup piece by piece.

File: lib/pbench/cli/agent/commands/cleanup.py [hunk 1]
```diff
@@ -19,7 +18,7 @@ def execute(self):
             "%s deprecated, will be removed in future release in favor of pbench-clear-results",
             self.name,
         )
-        run_command("pbench-clear-results")
+        os.execvp("pbench-clear-results")
```
Comment (dbutenhof) [inline lib/pbench/cli/agent/commands/cleanup.py:21]: So, `os.execlp("pbench-clear-results")` would have been better since you're not using an argv.

**Review by portante — COMMENTED** [review]
Okay, this now works.  Was missing an argument.  Added the output to the assertion to help debug.

File: lib/pbench/cli/agent/commands/cleanup.py (hunk 1 above)
Comment (webbnh) [reply lib/pbench/cli/agent/commands/cleanup.py]: Dang it, Peter!...I told you to change that!  🤪

**Review by webbnh — APPROVED** [review]
👍

File: lib/pbench/cli/agent/commands/cleanup.py [hunk 2]
```diff
@@ -19,7 +18,7 @@ def execute(self):
             "%s deprecated, will be removed in future release in favor of pbench-clear-results",
             self.name,
         )
-        run_command("pbench-clear-results")
+        os.execlp("pbench-clear-results", "pbench-clear-results")
```
Comment (dbutenhof) [inline lib/pbench/cli/agent/commands/cleanup.py:21]: OK, that's uglier than I hoped; but, yeah "path" and "arguments" and we need `argv[0]` to be set manually? Blech. 😬

**Review by dbutenhof — APPROVED** [review]

File: lib/pbench/cli/agent/commands/cleanup.py (hunk 2 above)
Comment (portante) [reply lib/pbench/cli/agent/commands/cleanup.py:21]: Still better than `run_command`!!!

Comment (webbnh) [reply lib/pbench/cli/agent/commands/cleanup.py:21]: Yep, I had both of those thoughts, too.

**Pull request merged**